tests/lib/rules/max-attributes-per-line.js

```diff
@@ -71,7 +71,7 @@ ruleTester.run('max-attributes-per-line', rule, {
   invalid: [
     {
       code: `<template><component name="John Doe" age="30" job="Vet" petname="Snoopy"></component></template>`,
-      errors: ['It has more than 3 attributes per line.']
+      errors: ['There are 4 in this line, but the maximum is 3.']
```

Comment:
Please add `attributes` keyword to the message :) So it says:
`There are 4 attributes in this line, but the maximum is 3.`